src/Spice86.Core/Emulator/InterruptHandlers/Dos/Ems/EmmHandle.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace Spice86.Core.Emulator.InterruptHandlers.Dos.Ems;
+
+/// <summary>
+/// Represents a handle for allocated EMS memory.
+/// </summary>
+public sealed class EmmHandle {
+    public ushort Pages { get; set; } = ExpandedMemoryManager.EmmNullHandle;
```

Comment:
Not sure what this represents, a handle ID?

Reply:
Yep

Reply:
It's been renamed to LogicalPage.